shared/test/test_assets.rb

```diff
@@ -1,221 +1,214 @@
-require 'mocha/mini_test'
-require_relative 'test_helper'
-
-require 'files_api'
-require 'channels_api'
+require_relative 'files_api_test_base' # Must be required first to establish load paths
+require_relative 'files_api_test_helper'
 require 'helpers/asset_bucket'
 require_relative 'spy_newrelic_agent'
 
-class AssetsTest < Minitest::Test
-  include SetupTest
+class AssetsTest < FilesApiTestBase
 
   def setup
-    @channels, @assets = init_apis
     # Ensure the s3 path starts empty.
-    delete_all_objects('cdo-v3-assets', 'assets_test/1/1')
-    @random = Random.new(0)
-  end
-
-  # Delete all objects in the specified path from S3.
-  def delete_all_objects(bucket, prefix)
-    raise "Not a test path: #{prefix}" unless prefix.include?('test')
-    s3 = Aws::S3::Client.new
-    objects = s3.list_objects(bucket: bucket, prefix: prefix).contents.map do |object|
-      { key: object.key }
-    end
-    s3.delete_objects(
-      bucket: bucket,
-      delete: {
-        objects: objects,
-        quiet: true
-      }
-    ) if objects.any?
+    delete_all_assets('assets_test/1/1')
   end
 
   def test_assets
-    channel_id = create_channel(@channels)
-
-    ensure_aws_credentials(channel_id)
+    channel_id = create_channel
+    api = FilesApiTestHelper.new(current_session, 'assets', channel_id)
```

Comment:
These two lines (or a slight variation) appear at the beginning of many different tests. It seems like you could extract definition of `@channel_id` and `@api` to `setup`, even if these don't get used in every test